scripts/recentActivitiesDataUpdate/projectDataUpdate.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+var monk = require('monk');
+var url = 'localhost:27017/mean-dev';
+var db = monk(url);
+
+var collection = db.get('recentactivities');
+
+collection.find({}).each(function(recentactivity) {
+  var contentUrl = recentactivity.contentUrl;
+  var documentUrl = recentactivity.documentUrl;
+  var id = recentactivity._id;
+
+  if( !contentUrl && !documentUrl ) {
+    recentactivity.contentUrl = contentUrl;
+    recentactivity.documentUrl = documentUrl;
+  }
+
+  if( contentUrl ) {
+    if(contentUrl.includes('https://projects.eao.gov.bc.ca')) {
```

Comment:
This is fine as the use case is pretty specific (and of limited scope). However, when processing strings it is generally good to use (if possible) regular expressions as they can make your life much easier!
In this case, your ```if/else``` statement could just be:
```recentactivity.contentUrl = recentactivity.contentUrl(/https:\/\/(www.)?\projects.eao.gov.bc.ca/, '');```

This will match both the public and non-public urls, and replace them with an empty string.
See [String.prototype.replace()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter) for reference.
https://regexr.com/ is one of the many online regex compilers that can be used to play around.